homeassistant/components/template/template_entity.py

```diff
@@ -417,30 +439,20 @@ def _async_template_startup(
     @callback
     def _async_setup_templates(self) -> None:
         """Set up templates."""
-        if self._availability_template is not None:
-            self.add_template_attribute(
-                "_attr_available",
-                self._availability_template,
-                None,
-                self._update_available,
-            )
```

Comment:
Is the availability template still prioritized / given special treatment over other templates?

Reply:
Yes, the rendering order and priority has not changed.  That's handled on line 426 of this file